packages/core/src/sass/app/modules/account-signup.scss

```diff
@@ -1,14 +1,6 @@
 /* @define account-signup; weak */
 .account-signup {
     padding: 1.2rem 1.2rem 0;
-    form {
-        @include mobile {
-            max-height: 36.5rem;
-        }
-    }
-    form:has(.account-signup__main) {
-        max-height: unset;
-    }
```

Comment:
'Sass/app/modules/account-signup.scss' was used is account-signup modal and password-selection modal. removing the limiting height of form results in auto sizing of the modal.